controller/persistence/posture_check_process.go

```diff
@@ -51,6 +52,13 @@ func (entity *PostureCheckProcess) LoadValues(_ boltz.CrudStore, bucket *boltz.T
 }
 
 func (entity *PostureCheckProcess) SetValues(ctx *boltz.PersistContext, bucket *boltz.TypedBucket) {
+
+	entity.Fingerprint = strings.ToLower(entity.Fingerprint)
```

Comment:
are fingerprints ever allowed to have whitespace or other delimiters in them? should that be normalized out if yes?

Reply:
Valid characters are `a-f, 0-9`. Could be normalized. I'll add a ticket for this because the work around is to do the right thing.

Reply:
https://github.com/openziti/edge/issues/400